playwright-tests/pages/components/stats.ts

```diff
@@ -0,0 +1,23 @@
+import { Locator, Page } from '@playwright/test';
+
+export class StatsBlock {
+  page: Page;
+  mainComponent: Locator;
+  input: Locator;
+  providerValue: Locator;
+  chainIdValue: Locator;
+  ethBalance: Locator;
+  stethBalance: Locator;
+  wstethBalance: Locator;
+
+  constructor(page: Page) {
+    this.page = page;
+    this.mainComponent = this.page.getByTestId('statsBlock');
```

Comment:
why its `mainComponent` but not `statsBlock` ?

Reply:
I think it is the correct name for the main locator of `class Stats`
Do you think I need to rename it to the `statsBlock`?